exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillReduceAggregatesRule.java

```diff
@@ -100,8 +108,13 @@ public void onMatch(RelOptRuleCall ruleCall) {
    */
   private boolean containsAvgStddevVarCall(List<AggregateCall> aggCallList) {
     for (AggregateCall call : aggCallList) {
-      if (call.getAggregation() instanceof SqlAvgAggFunction
-          || call.getAggregation() instanceof SqlSumAggFunction) {
+      SqlAggFunction sqlAggFunction = call.getAggregation();
+      if(sqlAggFunction instanceof DrillCalciteSqlWrapper) {
```

Comment:
Yeah,... I see the point. But this is necessary to still have wrapper here. 

For example, the type of avg(integer_type_col) in Calcite is defined as Integer. Thus, say, if the wrapper is removed somehow before this rule is fired, Calcite will complain the type it sees here (Integer) does not match with that (Double) we inferred before.